src/internal/svg/addSvg.tsx

```diff
@@ -11,9 +11,10 @@ export const addSvg = (
     fill="none"
     xmlns="http://www.w3.org/2000/svg"
   >
+    <title>Add SVG</title>
```

Comment:
this is what is read when accessibility settings are on, so we prefer to make the title "nicer". Recommend keeping this as "add"